snapcraft/cli/_runner.py

```diff
@@ -0,0 +1,78 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
```

Comment:
2018

Reply:
the changes here are negligible enough to not change the copyright